### PR TITLE
Snowflake db support

### PR DESCRIPTION
**Pull request by mbobrovsky**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -6,6 +6,7 @@
 # Folders
 _obj
 _test
+.idea
```
Comment (BenFradet) [inline .gitignore]: Can we remove this? This should belong to your [global gitignore](https://help.github.com/articles/ignoring-files/#create-a-global-gitignore).

File: .travis.yml [hunk 2]
```diff
@@ -29,7 +29,7 @@ deploy:
   provider: script
   script: release-manager --config .release.yml --check-version --make-version --make-artifact --upload-artifact
   on:
-    condition: '"${TRAVIS_GO_VERSION}" == "1.7.4"' 
+    condition: '"${TRAVIS_GO_VERSION}" == "1.9.2"'
     tags: true
```
Comment (alexanderdean) [inline .travis.yml]: ^^ These changes belong in #112

File: .travis.yml [hunk 3]
```diff
@@ -3,8 +3,8 @@ sudo: false
 
 language: go
 go:
-  - 1.6.4
-  - 1.7.4
+  - 1.9.2
+  - 1.8.5
```
Comment (alexanderdean) [inline .travis.yml]: Wrong order, lower to higher (and the additions belong in #112).

File: Godeps/Godeps.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
 	"ImportPath": "github.com/snowplow/sql-runner",
-	"GoVersion": "go1.7",
-	"GodepVersion": "v76",
+	"GoVersion": "go1.9",
```
Comment (alexanderdean) [inline Godeps/Godeps.json]: Really - did bumping to 1.9 not change any of the deps that are vendored in?

Comment (mbobrovsky) [reply Godeps/Godeps.json]: No, I didn't. You probably missed something.  There are some new deps in vendor.

File: .travis.yml (hunk 3 above)
Comment (mbobrovsky) [reply .travis.yml]: But it's travis.yml. You wrote about it in #110.

File: .travis.yml (hunk 2 above)
Comment (mbobrovsky) [reply .travis.yml]: But it's travis.yml. You wrote about it in #110.  Did I understand you wrong?

File: .gitignore (hunk 1 above)
Comment (mbobrovsky) [reply .gitignore]: Yes, my fault...

File: Godeps/Godeps.json (hunk 4 above)
Comment (alexanderdean) [reply Godeps/Godeps.json]: Not in this commit there aren't: https://github.com/snowplow/sql-runner/pull/115/commits/458abf70462e48a9ec67d32293b18625d946d6c1

File: .travis.yml (hunk 2 above)
Comment (alexanderdean) [reply .travis.yml]: Delete the rows in one commit, add the new rows in the other commit.

**Comment by mbobrovsky** [comment]
@alexanderdean @BenFradet Does it work for you? Does it look like you wanted?

**Comment by BenFradet** [comment]
LGTM 👍

**Comment by alexanderdean** [comment]
Yes - this is ready for UA. Thanks @mbobrovsky

**Comment by alexanderdean** [comment]
UA passed, cherry-picked into the release branch, closing.

Thanks so much @mbobrovsky! Good work.

**Pull request closed without merging**